eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -76,12 +76,19 @@ jobs:
         Linux:
           OSName: "Linux"
           OSVmImage: "ubuntu-16.04"
-        Windows:
+          TargetFramework: netcoreapp2.1
+        Windows_NetCoreApp:
           OSName: "Windows"
           OSVmImage: "vs2017-win2016"
+          TargetFramework: netcoreapp2.1
+        Windows_NetFramework:
+          OSName: "Windows"
+          OSVmImage: "vs2017-win2016"
+          TargetFramework: net461
```

Comment:
OK Changing this to TargetFramework broken the build, sorry about that. The reason it broken the build is because it ended up setting TargetFramework as an environment variable and msbuild picked that up and try to build some of the source projects, because of the project references from the tests, using that as the target framework. Instead lets name this TestTargetFramework which should hopefully avoid the collision with msbuild.